src/helper/storage/handler.js

```diff
@@ -5,7 +5,14 @@ var Warn = require('../warn');
 
 function StorageHandler() {
   this.warn = new Warn({});
-  this.storage = windowHandler.getWindow().localStorage || new CookieStorage();
```

Comment:
The PR name is use cookiestorage when local not available. It already did this though? This is a fix to catch errors no?

Reply:
renamed